src/chrome/content/rules/Madison-Logic.xml

```diff
@@ -6,57 +6,21 @@
 
 	Problematic domains:
 
-		- (www.)dinclinx.com	(mismatched, CN: *.madisonlogic.com)
-
-		- (www.)madisonlogic.com
-
-			- Data differ
-			- $ redirects to Login.aspx
-			- css/ & graphics/ 404
-
-
-	Partially covered domains:
-
-		- (www.)madisonlogic.com
-
+		- (www|blog.)madisonlogic.com	SSL certificate problem: unable to get local issuer certificate
+		  (Hosting by wpengine.com)
```

Comment:
Since the hosting information might change over time, we don't include this information in the ruleset comment. Would you remove this line ?

Reply:
This indicates who to bug about getting the ssl certs fixed, and it is more likely that the cert problem will be fixed before they change hosting provider, and the entire entry would then be removed from the Problematic hosts entry.

Anyways, happy to follow maintainer directions.